runProcessor.sh

```diff
@@ -0,0 +1,42 @@
+echo "+-----------------------------------------------------------------+"
+echo "| Enter the number that corresponds to your chosen network:       |"
+echo "| 1. interlay                                                     |"
+echo "| 2. kintsugi                                                     |"
+echo "| 3. interlay-testnet                                             |"
+echo "| 4. kintsugi-testnet                                             |"
+echo "+-----------------------------------------------------------------+"
+
+read choice
+
+if [ $choice == 1 ]; then
+  docker-compose up -d db-interlay
+elif [ $choice == 2 ]; then
+  docker-compose up -d db-kintsugi
+elif [ $choice == 3 ]; then
+  docker-compose up -d db-interlay-testnet
+elif [ $choice == 4 ]; then
+  docker-compose up -d db-kintsugi-testnet
+else
+  echo "Invalid choice. Options are 1, 2, 3, and 4"
+  exit 1
+fi
+
+if [ -f ~/.nvm/nvm.sh ]; then
```

Comment:
I wouldn't bother with all this additional logic. We can assume nvm is installed, same as we assume that yarn is installed.
They are part of the prerequisites to work with the package.

Reply:
Ah ok I will make these changes in the pr for the script and delete this file for this branch to keep this pr strictly for vault related code.